cookiecutter.json

```diff
@@ -7,5 +7,5 @@
     "project_version": "0.1.0",
     "short_description": "Short Project Description",
     "service_port": 5000,
-    "database": ["posgres", "elasticsearch", "memcached", "neo4j"]
+    "database": ["postgres", "elasticsearch", "memcached", "neo4j"]
```

Comment:
c.f. [juvenal .globality/build.json](https://github.com/globality-corp/juvenal/blob/develop/.globality/build.json#L3) which does use correct value as validation that we can safely change it in the cookiecutter